examples/web/RunAnywhereAI/src/styles/components.css

```
@@ -306,162 +552,504 @@
+  justify-content: center;
+  border: none;
+  border-radius: var(--radius-full);
+  background: var(--bg-tertiary);
+  color: var(--text-secondary);
+  cursor: pointer;
+}
+
+.composer-status-pill button svg {
+  width: 15px;
+  height: 15px;
+  color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)